NewDawn/NewDawn/ViewControllers/ChatPageViewControllers/ChatPageViewController.swift

```diff
@@ -88,6 +88,26 @@ class ChatPageTableViewModel: NSObject, UITableViewDelegate, UITableViewDataSour
         let cell = tableView.dequeueReusableCell(withIdentifier: "chatCell", for: indexPath) as! ChatCell
         // Get the end user's information
         if let user_id = currentMessageResponse[END_USER_ID] as? Int, let firstName = currentMessageResponse[END_USER_FIRSTNAME] as? String, let lastName = currentMessageResponse[END_USER_LASTNAME] as? String, let imageURL = currentMessageResponse[END_USER_IMAGE_URL] as? String {
+            LoginUserUtil.fetchLoginUserProfile() {
```

Comment:
pass `readLocal = false`

Reply:
similar to https://github.com/new-dawn/new_dawn_ios_playground/blob/master/NewDawn/NewDawn/ViewControllers/AppLandingViewController.swift#L17